.azure-pipelines.yml

```diff
@@ -42,9 +42,8 @@ stages:
         tags: |
           ort
 
-  # TODO: Add a separate job to run detekt to prevent running the matrix build if there are detekt errors?
-  #       Downside is that this requires separate cloning. Alternative is to run detekt only on one OS, or to run detekt
-  #       in a separate parallel job for early feedback, and disable detekt in the other jobs.
+  - template: ci/detekt.yml
```

Comment:
Since the new `ci` directory is only used for Azure Pipelines (currently at least), should we maybe give it a more specific name? Otherwise user might gets confused about how / if this related to e.g. the `Jenkinsfile`.

Reply:
I'll do that in a later PR so that this can be merged now and we get rid of the failing builds. What name would you prefer? "azure" or ".azure"? Or something else?

Reply:
Maybe just straight ".azure-pipelines"?